Scripts/static-analyzer/Sources/Analyzer/third-party/CompilerArgumentsExtractor.swift

```diff
@@ -0,0 +1,132 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2020 Realm Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+ */
```

Comment:
How do you feel about adding a link to the source?

```suggestion

 Sourced from: 
 https://github.com/realm/SwiftLint/blob/1f65377f9c0809edf035ede92d704e876e2cb59b/Source/swiftlint/Helpers/CompilerArgumentsExtractor.swift
 */
```

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21320/commits/ac79af7dc44395ed3905e6e5f6efc7d9e2e534ac. I plan to remove these two copied files with our own implementation. The `CompilerInvocations` type would be the replacement for this file.